fail2ban/protocol.py

```diff
@@ -55,6 +55,7 @@
 ["start <JAIL>", "starts the jail <JAIL>"], 
 ["stop <JAIL>", "stops the jail <JAIL>. The jail is removed"], 
 ["status <JAIL>", "gets the current status of <JAIL>"],
+["status <JAIL> extended", "gets the current status of <JAIL> with extended info"],
```

Comment:
I was thinking having the values all line up with each other, so someone could tie them together if they wanted to (match IP to RIR from parsing the output).

I think an overhaul/expansion of `fail2ban-client status` would be a good thing, but I'd rather not expand this enhancement to include it (which seems okay with you from your latest comment).